src/Files.Uwp/Helpers/UIHelpers.cs

```diff
@@ -94,7 +95,8 @@ public static async Task<BitmapImage> GetIconResource(int index)
 
         private static async Task<IEnumerable<IconFileInfo>> LoadSidebarIconResources()
         {
-            const string imageres = @"C:\Windows\System32\imageres.dll";
+            string systemRoot = Environment.GetEnvironmentVariable("Systemroot");
```

Comment:
Can you add this path to `CommonPaths`? https://github.com/files-community/Files/blob/main/src/Files.Uwp/Helpers/CommonPaths.cs

Reply:
Done